protobuf/sampling.proto

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2017 Uber Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+package jaegertracing.protobuf.sampling_manager;
+
+import "jaeger.proto";
+
+message ProbabilisticSamplingStrategy {
+    double sampling_rate = 1;
+}
+
+message RateLimitingSamplingStrategy {
+    int32 max_traces_per_seconds = 1;
+}
+
+message OperationSamplingStrategy {
```

Comment:
this can be embedded in PerOperationSamplingStrategy, correct?

Reply:
Guess so. I just copied this from Thrift. Then again, I suppose Thrift didn't support nesting.